src/docs/product/issues/issue-details/performance-issues/n-one-queries.mdx

```diff
@@ -64,3 +64,34 @@ def books(request):
 Django will `JOIN` the tables ahead of time, and preload the author information. That way, calling `book.author.name` does not need to make an extra query. Instead of a long waterfall, there is a single `SELECT` query:
 
 ![Solved N+1 queries in an example application](n-plus-one-queries-after.png)
+
+## MN+1 Queries
+
+<Include name="early-adopter-note.mdx" />
+
+Sometimes an application might make multiple queries in a loop, or make a query along with performing some other repetitive activities in a loop. To expand on the previous example:
+
+```python
+from django.http import HttpResponse
+
+def books(request):
+    books = Book.objects.select_related("author").all()[:10]
+    book_list = [book.title + " by " + book.author.name + ", " + book.publisher.name for book in books]
+    return HttpResponse((", ").join(book_list))
+```
+
+This code will perform two database queries for each iteration of the loop over books: one to fetch the author, and another to fetch the publisher. Since there are two queries in each iteration of the loop, we could call this a 2N+1 issue. Sentry will find and report these issues, too. They are categorized in the application as N+1 Queries, but may involve multiple repeating spans. MN+1 queries can be solved using similar strategies to N+1s, as [described above](#example).
+
```

Comment:
> Can we use N+1 queries, N+1 issues, and N+1s interchangeably?

* Colloquially (outside of Sentry), these are typically called N+1 DB queries, N+1 queries, or just N+1s. Once we establish the context that we're dealing with N+1 DB queries (as opposed to N+1 API calls or some other N+1 pattern), I think N+1 is reasonable as a shorthand.
* The label we've applied in the UI for issues of this type is "N+1 Query".
* Internally we've tried to avoid the word "issue" unless we actually mean an Issue (in the Sentry sense). We've been using the word "problem" as a substitute when "issue" would be ambiguous. (Sentry detects a performance problem in your code, and creates a Performance Issue to inform you and track its resolution).

I didn't do a particularly good job of following the above when writing the updates to this doc however 😅 

Does that make sense as a baseline?

> Does it make sense to say; "solving an N+1 issue?"

I think so, with the caveat above that "issue" can be confusing. If we mean a problem identified by Sentry (i.e. there exists an Issue for it) then it's fine; if we mean solving an issue in general, "solving an N+1 problem" might be less ambiguous.